README.md

```diff
@@ -6,15 +6,25 @@ You can find us on Twitter [@MozillaReality](https://twitter.com/mozillareality)
 
 [![Task Status](https://github.taskcluster.net/v1/repository/MozillaReality/FirefoxReality/master/badge.svg)](https://github.taskcluster.net/v1/repository/MozillaReality/FirefoxReality/master/latest) [Build results](https://github.taskcluster.net/v1/repository/MozillaReality/FirefoxReality/master/latest)
 
-## Bleeding-edge developer APKs
+## Download developer APKs (debug builds)
+
+**[Download](https://fxr.glitch.me/)** [bleeding-edge](https://github.com/MozillaReality/FirefoxReality/commits/master) developer (_debug_) APKs for Firefox Reality:
+
+- [Oculus Go (`oculusvr`)](https://fxr.glitch.me/oculusvr)
+- [HTC VIVE Focus & Pico G2 (`wavevr`)](https://fxr.glitch.me/wavevr)
+- [Google Daydream (`googlevr`)](https://fxr.glitch.me/googlevr)
+- [Snapdragon VR (`svr`)](https://fxr.glitch.me/svr)
+- [No API / Android Emulator for PC (`noapi`)](https://fxr.glitch.me/noapi)
```

Comment:
if y'all can think of a clearer description of `noapi`, I'll update this

Reply:
Maybe "No VR device API"? Up to you.